api.ts

```diff
@@ -783,14 +783,11 @@ export type GenericSyncUpdate = SyncUpdate<any, any, any>;
  * Type definition for a single update result returned by a sync table update function.
  * @hidden
  */
-export interface SyncUpdateSingleResult<
+export type SyncUpdateSingleResult<
   K extends string,
   L extends string,
   SchemaT extends ObjectSchemaDefinition<K, L>,
-> {
-  newValue?: ObjectSchemaDefinitionType<K, L, SchemaT>;
-  error?: Error;
-}
+> = ObjectSchemaDefinitionType<K, L, SchemaT> | Error;
```

Comment:
Just for my understanding, does `ObjectSchemaDefinitionType` refer to "an object that fulfills the given schema"?

Reply:
Yep - it's the same type we return from 1-way sync table pull requests.